# Study 869 — 52-Week-High Breakout Drift 🚀

**When a stock closes at a *fresh 52-week high*, does it drift up — or fade?**

This is the **event** of a new 52-week-high *breakout*, not George-Hwang **nearness** to
the high (that is study 236). Two stories are in tension: **breakout momentum** (a
decisive break above the old high releases pent-up demand, the name keeps running) versus
**resistance / anchoring** (the salient 52-week high is a ceiling, so the break fades). We
flag every fresh-52w-high day point-in-time on a liquid US cross-section
(2010-01-04 → 2026-06-30, 50 names) and measure the forward 5- and 20-day
return of a long-just-broke-out book vs the rest.

*Numbers below are the frozen headline (`docs/results.md`, fingerprint
`357fd262912f`); the live cells run the fast synthetic control. Survivorship:
current-membership mega-caps — survivors over-print new highs, so magnitudes are an upper
bound.*


## 1. The idea in one picture

A stock spends a year below some old peak, then one day *closes above it* for the first time in 52 weeks. Traders call this a **breakout**. The bullish read: the old high was a lid, and breaking it releases pent-up demand — buy new highs, they beget more highs. The bearish read: the 52-week high is a **resistance** level where anchored sellers dump and the move **fades**. Which one does the tape actually show over the next week and month?

In [1]:
R = dict(h5_spread=7.21, h5_t_nw=1.14, h5_brk=31.14, h5_rest=23.93,
         h20_spread=21.64, h20_t_nw=1.2, h20_brk=131.09, h20_rest=109.45)
print('FORWARD 5-DAY  : breakout book %+.2f bps vs rest %+.2f bps -> spread %+.2f (NW t=%+.2f)'
      % (R['h5_brk'], R['h5_rest'], R['h5_spread'], R['h5_t_nw']))
print('FORWARD 20-DAY : breakout book %+.2f bps vs rest %+.2f bps -> spread %+.2f (NW t=%+.2f)'
      % (R['h20_brk'], R['h20_rest'], R['h20_spread'], R['h20_t_nw']))

FORWARD 5-DAY  : breakout book +31.14 bps vs rest +23.93 bps -> spread +7.21 (NW t=+1.14)
FORWARD 20-DAY : breakout book +131.09 bps vs rest +109.45 bps -> spread +21.64 (NW t=+1.20)


## 2. Is the sort just lucky? A live synthetic control

We plant a breakout-drift in a seeded toy world (`edge>0`: names that just broke out get extra forward drift) and check the detector recovers it — and stays *silent* on the null (`edge=0`: fresh highs still happen but predict nothing). No network.

In [2]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from breakout_high import data, strategy as st
null = st.synthetic_detect(data.synthetic_panel(edge=0.0, seed=869, n_assets=40, n_days=1200))
planted = st.synthetic_detect(data.synthetic_panel(edge=0.0015, seed=869, n_assets=40, n_days=1500))
print('null world   : spread NW t = %+.2f  (should be ~0)' % null['t_nw'])
print('planted world: spread NW t = %+.2f  (should light up)' % planted['t_nw'])

null world   : spread NW t = +1.12  (should be ~0)
planted world: spread NW t = +9.62  (should light up)


## 3. The honest verdict — a drift in the *right direction* that doesn't clear the bar

On this liquid mega-cap tape the fresh-breakout name **does** out-earn the rest — the sign is **breakout momentum, not fade**: **+7.21 bps** over 5 days, **+21.64 bps** over 20 days. But the honest, overlap-corrected Newey-West *t* is only **+1.14** / **+1.20** — below the |*t*| = 2 bar. The permutation placebo puts it at p ≈ 0.10–0.08, and the whole effect is a **2018-2026** phenomenon (2010-2017 is flat-to-negative). Directionally there, statistically absent. **Signal: Weak**, and once you pay to trade it (**-13.48 bps net** at 5 bps/side over 5 days) — **Tradability: Mirage**.